assets/src/edit-story/utils/hexToRGBA.js

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Convert 6-digit hex to rgba value.
+ *
+ * @param {string} hex 6-digit hex value.
+ * @param {number} opacity Opacity in %.
+ */
+function hexToRGBA(hex, opacity = 100) {
```

Comment:
Does it do anything different than this one?

```js
import { rgba } from 'polished';
```

Reply:
🤦‍♀ I suppose not, I just like reinventing the wheel.

Thanks for noting that, will reuse!

Reply:
Changed in 1bfbe0e